Servers/UI/OJS.Servers.Ui/ClientApp/src/components/submissions/test-run-details/TestRunDetails.tsx

```diff
@@ -52,7 +53,29 @@ const TestRunDetails = ({ testRun }: ITestRunDetailsProps) => {
         return testRunText;
     }, [ testRun ]);
 
-    const renderTimeAndMemoryUsed = useCallback(() => (
+    const renderLabel = useCallback(
+        () => {
+            const result = testRun.resultType.toLowerCase();
+            const type = result === 'correctanswer'
+                ? LabelType.success
+                : result === 'wronganswer'
```

Comment:
'wronganswer' can be extracted as constant

Reply:
Done.